omod/pom.xml

```diff
@@ -141,8 +141,14 @@
 		<dependency>
 			<groupId>com.fasterxml.jackson.core</groupId>
 			<artifactId>jackson-databind</artifactId>
+		</dependency>         
+		<dependency>
+			<groupId>org.openmrs.api</groupId>
+			<artifactId>openmrs-api</artifactId>
+			<version>${openMRSVersion}</version>
+			<type>test-jar</type>
+			<scope>test</scope>
```

Comment:
For what use-case is this change made?

Reply:
I followed this wiki page- https://wiki.openmrs.org/plugins/servlet/mobile?contentId=3346858#content/view/3346858